zuul.d/projects.yaml

```diff
@@ -391,3 +391,10 @@
     3pci-check:
       jobs:
         - noop
+
+- project:
+    name: github.com/ansible/workshops
+    default-branch: devel
+    3pci-check:
+      jobs:
+        - ansible-tox-linters
```

Comment:
Are you okay with this job as voting? That means, we'll report job failures right away.

Reply:
Yes - as its already voting via Jenkins privately - this will just make it public for other contributor to be able to check whats wrong